requirements-optional.txt

```diff
@@ -6,6 +6,7 @@ pre-commit==3.3.3
 
 # For generating documentation.
 sphinx==7.2.3
+sphinx==7.2.5
```

Comment:
Can you rebase your branch and resolve conflicts, or remove this ?

Reply:
rebased, thanks!